alignak/shinken_import_hook.py

```diff
@@ -1,5 +1,5 @@
-
 # -*- coding: utf-8 -*-
```

Comment:
Careful. I got the encoding line at line n3/4 previously and it was not working. I don't know if it's in the first line it is stil OK